sirmordred/task_collection.py

```diff
@@ -106,7 +106,12 @@ def execute(self):
             repo, repo_labels = self._extract_repo_labels(self.backend_section, repo)
             p2o_args = self._compose_p2o_params(self.backend_section, repo)
             filter_raw = p2o_args['filter-raw'] if 'filter-raw' in p2o_args else None
+            no_collection = p2o_args['filter-no-collection'] if 'filter-no-collection' in p2o_args else None
```

Comment:
Use a `.get()` instead.

Reply:
@quan can you apply the same comment to the `filter-raw` variable?